packages/cli-lib/lib/read.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+const path = require('path');
+
+const STAT_TYPES = {
+  FILE: 'file',
+  SYMBOLIC_LINK: 'symlink',
+  DIRECTORY: 'dir',
+};
+
+const generateFilePromise = (dir, file) => {
```

Comment:
Maybe rename `getFileInfoAsync` to make the purpose clearer?

Reply:
Good call, done in b12426c